tools_for_todoist/services/calendar_to_todoist.py

```diff
@@ -90,19 +92,39 @@ def _process_cancelled_event(self, calendar_event):
             todoist_item = self.todoist.get_item_by_id(todoist_id)
             if todoist_item is not None:
                 self.todoist.delete_item(todoist_item)
+    
+    def _process_updated_event(self, calendar_event):
+        print('Updating event|', calendar_event)
+        todoist_item = self.todoist.get_item_by_id(_todoist_id(calendar_event))
+        if todoist_item is None:
+            print('Invalid event??')
+            return None
+        next_occurence = calendar_event.next_occurrence()
+        if next_occurence is None:
+            self.todoist.delete_item(todoist_item)
+            return None
+        todoist_item.set_due(next_occurence, calendar_event.recurrence_string())
+        todoist_item.save()
+        return calendar_event, todoist_item
 
     def _google_calendar_sync(self):
         sync_result = self.google_calendar.sync()
-        new_event_item_links = []
+        to_be_synced_event_item_links = []
 
         for calendar_event in sync_result['created']:
             new_event_item_link = self._process_new_event(calendar_event)
             if new_event_item_link is not None:
-                new_event_item_links.append(new_event_item_link)
+                to_be_synced_event_item_links.append(new_event_item_link)
 
         for calendar_event in sync_result['cancelled']:
             self._process_cancelled_event(calendar_event)
-        return sync_result, new_event_item_links
+        
+        for calendar_event in sync_result['updated']:
+            updated_event_item_link = self._process_updated_event(calendar_event)
+            if updated_event_item_link is not None:
+                to_be_synced_event_item_links.append(updated_event_item_link)
```

Comment:
why are you adding these here? The calendar event already has the appropriate link to the todoist ID? Please revert the rename and remove this.